Packs/CommonTypes/IncidentFields/incidentfield-Device_Id.json

```diff
@@ -0,0 +1,37 @@
+{
+	"associatedToAll": true,
+	"associatedTypes": ["Tripwire File Change"],
+	"breachScript": "",
+	"caseInsensitive": true,
+	"cliName": "deviceid",
+	"closeForm": false,
+	"columns": null,
+	"content": true,
+	"defaultRows": null,
+	"description": "Device Id",
+	"editForm": true,
+	"fieldCalcScript": "",
+	"group": 0,
+	"hidden": false,
+	"id": "incident_deviceid",
+	"isReadOnly": false,
+	"locked": false,
+	"name": "Device Id",
+	"neverSetAsRequired": false,
+	"ownerOnly": false,
+	"placeholder": "",
+	"required": false,
+	"script": "",
+	"selectValues": null,
+	"sla": 0,
+	"system": false,
+	"systemAssociatedTypes": null,
```

Comment:
```suggestion
	"systemAssociatedTypes": ["Tripwire File Change"],
```

Reply:
It should be null for new incident fields, according to david, only for modified incident fields this key should be the list of associated types.